addons/account/models/company.py

```diff
@@ -98,6 +98,9 @@ class ResCompany(models.Model):
 
     invoice_is_email = fields.Boolean('Email by default', default=True)
     invoice_is_print = fields.Boolean('Print by default', default=True)
+    account_credit_limit = fields.Boolean(string='Credit Limit', help='Enables the use of credit limit on Partners.')
```

Comment:
Either you have a default value for the company or you don't, no need to store this either